packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -83,6 +68,20 @@ const Signup = ({
   ] = useState<boolean>(false);
   const navigate = useNavigate();
   const location = useLocation();
+  const [client, setClient] = useState<MozServices | undefined>(undefined);
+
+  useEffect(() => {
+    if (isOAuthIntegration(integration)) {
+      const clientId = integration.getService();
+      if (isClientPocket(clientId)) {
```

Comment:
Perhaps this logic should go in the container? This could let us make make the client ids (and client state in general more data driven).